app/serializers/site_status_serializer.rb

```diff
@@ -26,12 +26,9 @@ def status
     object.status_before_type_cast
   end
 
-  # rubocop:disable Style/DateTime
   def course_open_date
-    # TODO applications_accepted_from should be a timestamp, not a string
-    DateTime.parse(object.applications_accepted_from).iso8601 if object.applications_accepted_from.present?
+    object.applications_accepted_from if object.applications_accepted_from.present?
```

Comment:
We can do away with the `if ...` guard, since we don't need to convert this anymore. The only possible values are Date or nil.

Reply:
Done